spock-core/src/main/java/org/spockframework/runtime/model/FeatureInfo.java

```diff
@@ -20,6 +20,9 @@ public class FeatureInfo extends SpecElementInfo<SpecInfo, AnnotatedElement> imp
   private final List<String> dataVariables = new ArrayList<>();
   private final List<BlockInfo> blocks = new ArrayList<>();
   private final List<IMethodInterceptor> iterationInterceptors = new ArrayList<>();
+  private final List<IMethodInterceptor> setupInterceptors = new ArrayList<>();
```

Comment:
The change looks good to me, but I guess the new interceptor types should be mentioned in the documentation.